test/jest/default-request-executor.test.js

```diff
@@ -144,7 +144,7 @@ describe('DefaultRequestExecutor', () => {
       requestExecutor.retryRequest = jest.fn();
     });
 
-    it('should defer to delayFetch for retry-able 429 responses, within max elapsed time', () => {
+    it('should defer to delayFetch if the request is not timed and not in a max-retry state', () => {
```

Comment:
In the context of this code, what is delayFetch? Had mentioned in the original comment, but what the code is actually testing is that it defers to *retryRequest*.